addons/onedrive/client.py

```diff
@@ -44,7 +43,7 @@ def folders(self, drive_id=None, folder_id=None):
             throws=HTTPError(401)
         )
         folder_list = resp.json()
-        logger.debug('folder_list:({})'.format(folder_list))
+        logger.debug(f'folder_list:({folder_list})')
```

Comment:
I'd like to keep it in at `debug` level. When debugging addons, provider responses are the first thing we look at. A lot of provider bugs are because of the response structure changes or misunderstandings.